Tests/Unit/Search/Metadata/Driver/XmlDriverTest.php

```diff
@@ -49,11 +49,12 @@ public function setUp()
 
         $this->reflectionClass = $this->prophesize(\ReflectionClass::class);
         $this->reflectionClass->getName()->willReturn('Sulu\Bundle\ExampleBundle\Entity\Example');
+        $this->reflectionClass->__toString()->willReturn('what the hell');
```

Comment:
Is that on purpose? Looks like something you just put there temporarily.

Reply:
ups will change the text but something in prophecy itself is do a __toString now.